### DISCUSSION TITLE
This Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by calakersky** [vercel/vercel#9233] [Help] [answered]
I have deployed my Nuxtjs app to Vercel, I don't know what's wrong.
![image](https://user-images.githubusercontent.com/66521798/212485876-a4b7d7c6-f937-4512-b441-83693dd43a1f.png)

**Comment by leerob** [comment, marked as answer]
Hey, have you checked out the error logs, linked on that page?

**Reply by calakersky** [reply]
![image](https://user-images.githubusercontent.com/66521798/212487767-6cc53ca9-a2e5-4958-bd51-285e1295eefe.png)
Show like this

**Reply by leerob** [reply]
Looks like this is an issue with your Nuxt application setup. If you run the build locally, stimulating production, you should likely see the same issue. Can you confirm?

**Reply by calakersky** [reply]
I'm build in local working normally

**Reply by leerob** [reply]
Are you running the production server locally? Have you tried what the error message suggests above with `serverFiles`?

**Reply by calakersky** [reply]
Now working, Thanks
